lib/jekyll-feed/generator.rb

```diff
@@ -33,6 +33,15 @@ def config
       @config ||= @site.config["feed"] || {}
     end
 
+    # Returns the metadata for the collection in the argument
+    def data_for_collection(name)
+      @site.config["collections"][name]
+    end
```

Comment:
Instead of extracting data from `@site.config`, I suggest utilizing existing metadata:

```ruby
    # Returns the metadata for the collection in the argument
    def data_for_collection(name)
      @site.collections[name].metadata
    end
```